host_modules/systemd_service.py

```diff
@@ -0,0 +1,44 @@
+"""Systemd service handler"""
+
+from host_modules import host_service
+import subprocess
+
+MOD_NAME = 'systemd'
+ALLOWED_SERVICES = ['snmp', 'swss']
```

Comment:
>ALLOWED_SERVICES [](http://example.com/codeflow?start=0&length=16)

Let's check all the requirements and give a full service list useful. Like lldp, telemetry, etc.

Reply:
Extended to include dhcp_relay, radv, restapi, lldp, and sshd. I did not include telemetry for now because telemetry docker may host gNOI server

Reply:
Is it useful to restart telemetry or rsyslog or pmon?

Reply:
I've extended to include pmon and rsyslog. Since gNOI server may run in telemetry docker, I do not include support for telemetry restart here

Reply:
Sure, we can place trust in client to request telemetry safely. I have added ability to restart telemetry